Lesson7/Task4.java

```diff
@@ -0,0 +1,39 @@
+package Lesson7;
+
+public class Task4 {
+    public  static  void  main(String  args[]) {
+        TV tv1 = new TV("Sony");
+        RemoteController rc1 = new RemoteController(tv1);
+
+        tv1.currentChannel();
+        tv1.powerOn();
+        tv1.powerOff();
+        tv1.setChannel(3);
+        tv1.currentChannel();
+        tv1.upVolume();
+        tv1.upVolume();
+        tv1.upVolume();
+        tv1.downVolume();
+        tv1.mute();
+
+        rc1.connectDevice(tv1);
+        rc1.disconnectDevice();
+        rc1.upVolume();
+        rc1.downVolume();
+        rc1.mute();
+        rc1.switchTheChannel(33);
+
+        rc1.checkIfDeviceConnected(tv1);
+        if (rc1.checkIfDeviceConnected(tv1) == true) {
+            System.out.println("Connected");
+        }
+            else {
+            System.out.println("Disconnected");
+            }
```

Comment:
This verification should be inside checkIfDeviceConnected method. You can create variable there such as isConnected == false;
then do some verification and inside if/else print message and set isConnected variable and after if/else return isConnected.